src/ontology/fbbt-edit.obo

```diff
@@ -118131,6 +118131,145 @@ relationship: develops_from FBbt:00004945 ! onion stage spermatid
+relationship: expresses http://flybase.org/reports/FBgn0036713 ! Mip
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-04-13T13:45:45Z
+
+[Term]
+id: FBbt:00058004
+name: adult anterior midgut class II enteroendocrine cell
+def: "Class II enteroendocrine cell localized in the anterior midgut of the adult. These cells express NPF and Dh31 in addition to Tachykinin (Chen et al., 2016; Guo et al., 2019)." [FlyBase:FBrf0232200, FlyBase:FBrf0244352]
+subset: cur
+intersection_of: FBbt:00110036 ! adult class II enteroendocrine cell
+intersection_of: part_of FBbt:00047154 ! adult anterior midgut
+relationship: expresses http://flybase.org/reports/FBgn0027109 ! NPF
+relationship: expresses http://flybase.org/reports/FBgn0032048 ! Dh31
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-04-13T13:46:57Z
+
+[Term]
+id: FBbt:00058005
+name: adult middle midgut class II enteroendocrine cell
+def: "Class II enteroendocrine cell localized in the middle midgut of the adult. These cells express NPF and Allatostatin B/Mip in addition to Tachykinin (Chen et al., 2016; Guo et al., 2019)." [FlyBase:FBrf0232200, FlyBase:FBrf0244352]
+subset: cur
+intersection_of: FBbt:00110036 ! adult class II enteroendocrine cell
+intersection_of: part_of FBbt:00110897 ! adult midgut region R3
+relationship: expresses http://flybase.org/reports/FBgn0027109 ! NPF
+relationship: expresses http://flybase.org/reports/FBgn0036713 ! Mip
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-04-13T13:48:08Z
+
+[Term]
+id: FBbt:00058006
+name: adult posterior midgut class II enteroendocrine cell
+def: "Class II enteroendocrine cell localized in the posterior midgut of the adult. These cells express NPF and Dh31 in addition to Tachykinin (Chen et al., 2016; Guo et al., 2019)." [FlyBase:FBrf0232200, FlyBase:FBrf0244352]
+subset: cur
+is_a: FBbt:00110207 ! enteroendocrine cell of posterior adult midgut epithelium
+intersection_of: FBbt:00110036 ! adult class II enteroendocrine cell
+intersection_of: part_of FBbt:00047155 ! adult posterior midgut
+relationship: expresses http://flybase.org/reports/FBgn0027109 ! NPF
+relationship: expresses http://flybase.org/reports/FBgn0032048 ! Dh31
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-04-13T13:49:22Z
+
+[Term]
+id: FBbt:00058007
+name: Dh31 cell of the embryonic/larval midgut
+def: "Enteroendocrine cell of the larval midgut that secretes the diuretic hormone Dh31 (FBgn0032048). These cells are particularly abundant in the anterior midgut but are also present in the middle and posterior midgut (Veenstra et al., 2009)." [FlyBase:FBrf0207758]
+subset: cur
+intersection_of: FBbt:00047150 ! larval enteroendocrine cell
+intersection_of: expresses http://flybase.org/reports/FBgn0032048 ! Dh31
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-04-14T08:32:51Z
+
+[Term]
+id: FBbt:00058008
+name: larval anterior middle midgut junction
```

Comment:
the way you have defined this it should probably be 'anterior-middle', though if you defined it as the anteriormost part of the middle midgut 'anterior middle' would make sense.